autogpt_platform/backend/backend/server/routers/v1.py

```diff
@@ -121,7 +130,7 @@ class DeleteGraphResponse(TypedDict):
 async def get_graphs(
     user_id: Annotated[str, Depends(get_user_id)],
     with_runs: bool = False,
-) -> list[graph_db.GraphMeta]:
+) -> Sequence[graph_db.GraphMeta]:
```

Comment:
Replaced all `list[T]` by `Sequence[T]` because `list[T]` is invariant and doesn't allow passing a `list[SUBCLASS_OF_T]` as `list[T]`.